### PR TITLE
Add deleting comments from article

### PR DESCRIPTION
**Pull request by jasonbarrows**

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -70,3 +74,9 @@ const apiPatch = (url, body) => {
     body: JSON.stringify(body),
   });
 };
+
+const apiDelete = (url) => {
+  return api(url, {
+    method: 'DELETE',
+  });
+};
```
Comment (itskatherine) [inline src/api/index.js:82]: Brill, added this function when you needed it and no sooner 🎉

File: src/components/CommentCard.jsx [hunk 2]
```diff
@@ -1,23 +1,41 @@
...
+        return prev.filter((stateComment) => {
+          return stateComment.comment_id !== comment.comment_id;
+        });
+      });
+    }).catch(() => {
+      setHasDeleteError(true);
+    }).finally(() => {
+      setIsDeleting(false);
+    });
+  };
```
Comment (itskatherine) [inline src/components/CommentCard.jsx:31]: Nice error handling here - could use a loading state for when the user has slow internet and presses the delete button to let them know deletion is in progress while they're waiting for the API response 🔍

**Review by itskatherine — COMMENTED** [review]
Great work Jason, clean UI and responsive design (I'm almost tired of typing that on your PRs haha) Once you've addressed my comments you're good to merge 👍

File: src/components/CommentCard.jsx (hunk 2 above)
Comment (jasonbarrows) [reply src/components/CommentCard.jsx:31]: Thanks Katherine, great suggestion!

**Pull request merged**